.github/workflows/shellcheck.yml

```diff
@@ -19,6 +19,4 @@ jobs:
     steps:
       - name: 'Check scripts in all directories'
         shell: bash
-        run: |
-          shopt -s globstar
-          shellcheck ./**/*.sh
+        run: shellcheck $(find -name "**.sh")
```

Comment:
This works as expected, as long as there is at least ONE `.sh` file. Otherwise `$(find -name "**.sh")` evaluates to nothing.

It may be worth having multiple lines in the script, and assign the result of find to a var, and check that it is not empty.

Alternatively, check the result of `find ... | grep .`.